src/content/blog/v1.0_release.md

```diff
@@ -0,0 +1,63 @@
+
+### A new set of Emulator models
+
+AutoEmulate v1.0.0 includes a new set of Emulator models with a variety of different architectures. Some of these are wrapped implementations from PyTorch, Sklearn and elsewhere, but we also have a number of custom implementations including ensemble models where we quantify uncertainty by looking at the distribution over outputs. The table below summarises the models available in this release:
+
+| Emulator | Implementation | Backend | Multi-output support |
+|----------|----------------|---------|----------------------|
+| GaussianProcessExact | Custom | PyTorch | True |
+| GaussianProcessExactCorrelated | Custom | PyTorch | True |
+| LightGBM | Wrapped | LightGBM | False |
+| SupportVectorMachine | Wrapped | Scikit-learn  | True |
+| RadialBasisFunctions | Wrapped | PyTorch | True |
+| RandomForest | Wrapped | Scikit-learn | True |
+| MLP | Custom | PyTorch | True |
+| EnsembleMLP | Custom | PyTorch | True |
+| EnsembleMLPDropout | Custom | PyTorch | True |
+
+### Bayesian model calibration
+
+Bayesian calibration estimates the probability distribution over input parameters given observed data, providing uncertainty estimates.
+Performing Bayesian calibration requires a fit emulator that predicts uncertainty (e.g., Gaussian Process) and observations associated with the simulator output. Check out our new [tutorial](<URL>) for more details on how to use this feature.
+
+### Upgraded features
+
+- **Simulator in the loop**: Some features in AutoEmulate such as *active learning* and *history matching* require the user to provide a simulator that can be run by AutoEmulate. AutoEmulate now supports easy integration of simulators into the emulation workflow. This is achieved through a subclassing mechanism of the *Simulator* base class, allowing the users to define a method to run their simulation, taking in the input parameters and returning the output variables. See our [custom simulations tutorial](<URL>) for more details.
+- **Active learning** is now a core feature of AutoEmulate. Active learning intelligently selects informative simulator evaluations to maximize emulator improvement with minimal computational cost. See our [active learning tutorial](<URL>) for more details.
+
+### Key API changes
+
+As with previous releases, AutoEmulate v1.0.0 can be used to quickly get an emulator from simulation inputs (X) and outputs (y). This now happens in a single step:
+
+```python
+ae = AutoEmulate(X, y)
+```
+
+This replaces the previous two-step process of `ae.setup(X, y)` and `ae.compare()`. Check out the [Quick Start Guide](<URL>) for more details on how to use the new API.
+
+## What's next
```

Comment:
I guess we should mention something about spatio-temporal data here (it's what we say in the Turing blog and it's always been the next step on our road map).

Reply:
Have added this but is fairly minimal, same as the main blog